### PR TITLE
feat(private): add vm subnet

### PR DESCRIPTION
**Pull request by mehalter**
This simply adds a subnet for VMs to use. I was going to hold off getting this merged in, but it's pretty useful in the development of the VM AMIs

### CODE REVIEW

File: capeinfra/swimlanes/private.py [hunk 1]
```diff
@@ -38,6 +38,17 @@ def __init__(self, name, *args, **kwargs):
             opts=ResourceOptions(parent=self),
         )
 
+        subnet_name = f"{name}-vm-subnet"
```
Comment (thecaffiend) [inline capeinfra/swimlanes/private.py:41]: we're probably going to end up with a few subnets. i'd say this one is geared toward compute (even more specifically analysis pipeline compute), so can we name it a little more specifically (the name, name variable and description)?

File: capeinfra/swimlanes/private.py [hunk 2]
```diff
@@ -38,6 +38,17 @@ def __init__(self, name, *args, **kwargs):
...
+        self.vm_subnet = aws.ec2.Subnet(
+            subnet_name,
+            cidr_block="10.0.1.0/24",
```
Comment (thecaffiend) [inline capeinfra/swimlanes/private.py:44]: this kinda feels it should be configurable. i won't hold up the merging, but pls make an issue to get that modded

File: capeinfra/swimlanes/private.py (hunk 1 above)
Comment (mehalter) [reply capeinfra/swimlanes/private.py]: Revised in latest commit, let me know what you think

File: capeinfra/swimlanes/private.py (hunk 2 above)
Comment (mehalter) [reply capeinfra/swimlanes/private.py:44]: Yeah, I'll open an issue for this

**Comment by thecaffiend** [comment]
:+1:

**Review by thecaffiend — APPROVED** [review]

**Pull request merged**